crates/fuel-gas-price-algorithm/src/v1.rs

```diff
@@ -239,6 +239,12 @@ impl AlgorithmUpdaterV1 {
             // gas prices
             self.update_exec_gas_price(used, capacity);
             self.update_da_gas_price();
+
+            // metadata
+            self.unrecorded_blocks.push(BlockBytes {
+                height,
+                block_bytes,
+            });
```

Comment:
I don't see how it is related to the change in the gas price analyser

Reply:
True, it's not related. It was something I discovered when working on this issue and decided to add a fix in the same PR.

Even though the PR description mentions this change, I should probably issue a separate ticket and create a different PR.

Would you like me to clean-up the situation for this case or you think we can merge as is?

Reply:
If @MitchTurner is okay of merging it in this PR, I'm also okay=)